examples/nlp/intent_detection_slot_tagging/joint_intent_slot_with_bert.py

```diff
@@ -99,6 +99,7 @@
     create_tb_writer=True,
     files_to_copy=[__file__],
     add_time_to_log_dir=True,
+    placement=nemo.core.DeviceType.CPU,
```

Comment:
why `placement=nemo.core.DeviceType.CPU` ?

Reply:
fixed that. I was testing on my laptop. It does not run on cpu without that.